libr/bin/format/elf/elf.c

```diff
@@ -2347,6 +2347,7 @@ void* Elf_(r_bin_elf_free)(struct Elf_(r_bin_elf_obj_t)* bin) {
 	return NULL;
 }
 
+//XXX: this function will break unless r_file_slurp is fixed
```

Comment:
I might work with this in next PR. Give me a day, if I can get it done, I will update this PR else the next PR. :)